refapp/SKWebApi/Skills/SystemPromptDefaults.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SKWebApi.Skills;
+
+internal static class SystemPromptDefaults
+{
+    internal const double TokenEstimateFactor = 2.5;
+    internal const int ResponseTokenLimit = 1024;
+    internal const int CompletionTokenLimit = 8192;
+    internal const double MemoriesResponseContextWeight = 0.3;
+    internal const double HistoryResponseContextWeight = 0.3;
+    internal const string KnowledgeCutoffDate = "1/1/2022";
```

Comment:
Please add a comment on the format of this date, DD/MM/YYYY or MM/DD/YYYY?

Reply:
Or use the ISO format? (YYYY-MM-DD)